docs/quickstart/quickstart.ipynb

```diff
@@ -133,7 +133,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.3"
+   "version": "3.7.10"
```

Comment:
For this issue we only want to change the ylabel and xlim, so you will want to undo this change.

Reply:
That's just metadata, the only way he can change that is to update his python

Reply:
Can this be editted in the file with a commit so that it will be reverted? I was under the impression that only the graph edits.

Reply:
It's ok. @kaimartin365 used the newest TARDIS environment which reverted Python to 3.7. The metadata always reflect the last Python version used to render that notebook, even when you clear all the notebook cells.